pkg/openstackprovisionserver/deployment.go

```diff
@@ -42,20 +42,19 @@ func Deployment(
 	startupProbe := &corev1.Probe{
 		// TODO might need tuning
 		TimeoutSeconds:      5,
-		PeriodSeconds:       3,
-		InitialDelaySeconds: 30,
+		PeriodSeconds:       10,
+        failureThreshold:    30,
```

Comment:
Sorry, my comment was using the YAML format for the field.  In Golang it should be `FailureThreshold`.

Reply:
Also, I think we can probably use a lower threshold.  The `30` I had was just from the example I found online.  Maybe `12`?   That would give the startup probe 2 minutes (12*10=120 seconds).  Does that seem like enough time?